database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
+* go
+  
+  [Download and install go](https://go.dev/doc/install)
+* psql
+  ```bash
+  sudo dnf install postgresql
+  psql --version
+  psql (PostgreSQL) 15.6
+  ```
+* DBeaver Community
+  
+  A SQL client is recommended to interact easily with the Database. 
+  [DBeaver Community](https://dbeaver.io/).
+
+## Manual deployment
+
+The PostgreSQL manual deployment consists in 2 parts: First, the deployment of the Database infrastructure and then, using a script, create a table in the Database and populate it.
+
+### Deploy the Database infrastructure
+
+* Create a ConfigMap with the Database details.
```

Comment:
Database should not be capitalized here.

```suggestion
* Create a ConfigMap with the database details.
```

I won't mark the rest of them in the document, but they should be updated as well.